samples/algorithms/chsh-game/README.md

```diff
@@ -67,8 +57,7 @@ Press Start in Visual Studio to run the sample.
 ## Manifest
 
 - [Game.qs](https://github.com/microsoft/Quantum/blob/main/samples/algorithms/chsh-game/Game.qs): Q# code implementing the game.
-- [host.py](https://github.com/microsoft/Quantum/blob/main/samples/algorithms/chsh-game/host.py): Python host program to call into the Q# sample.
-- [Host.cs](https://github.com/microsoft/Quantum/blob/main/samples/algorithms/chsh-game/Host.cs): C# code to call the operations defined in Q#.
+- [Program.qs](https://github.com/microsoft/Quantum/blob/main/samples/algorithms/chsh-game/Program.qs): Q# program entry point for the example.
```

Comment:
It looks like this is the link that's flagging the error in link checking, but it should be valid once your PR is completed such that I think we should be OK to ignore that optional check.

Reply:
I changed the links to be relative, which fixed the error.